### DISCUSSION TITLE
Looking for ways to use wz_mini_hacks to enable/disable recording without rebooting the camera

### DISCUSSION THREAD

**Discussion by Sadie33** [gtxaspec/wz_mini_hacks#496] [General]
Is it possible to enable/disable camera recording without having to reboot the camera?

When wz_mini_hacks is running, can you use the Wyze App with rules to enable/disable recording without rebooting the camera?

I have several Wyze Cam 3 and one Wyze Cam 2 running wy_mini_hacks. I have some of the V3 cameras outside recording wildlife continuously, using NFS to store the recordings my my NAS. These camera seldom need to reboot and are working fine.

I use the other V3 (and one V2) for surveillance. I'm using Node Red to detect when we're home or away. When we're away, Node Red will trigger a shell script running on a Raspberry Pi to upload a version of wze_mini.conf and .user_config to enable recording. Once uploaded, I reboot the camera to start recording. This is where things occasionally go awry. Most of the time the camera will reboot via wz_mini_hacks. However, sometime it doesn't and boots the stock firmware. If recording was enabled, it uses the SD-CARD to store recordings. Power cycling the camera (sometimes more than once) eventually boots via wz_mini_hacks successfully, but sometimes it doesn't.

I figure, if I could enable/disable recording without having to reboot the camera, I could avoid this issue.

**Comment by sideup66** [comment]
Is your event recording being done through RTSP or are you using standard wyze software to do this?
